.gitignore

```diff
@@ -34,5 +34,15 @@ node_modules
 # Optional REPL history
 .node_repl_history
 package-lock.json
+<<<<<<< HEAD
+<<<<<<< HEAD
 .vscode
-.env
\ No newline at end of file
+.env
+=======
+.nyc_output
+coverage
+>>>>>>> CH(Coveralls): integrate coveralls with Authors Haven
```

Comment:
Fix the merge conflicts here

Reply:
Conflict fixed

Reply:
It's not fixed yet. Can still see `<<<< HEAD` as well as your commit hashes.

Reply:
> It's not fixed yet. Can still see `<<<< HEAD` as well as your commit hashes.

Oh, I didn't notice that of .gitignore. Fixing it asap